### PR TITLE
+ Button

### PR DESCRIPTION
**Pull request by Hoshi-Okami**
Like mission configuration, a + button was added to allow the user to add multiple SADE zones. The max amount of SADE zones at the moment is 10.

### CODE REVIEW

**Comment by Hoshi-Okami** [comment]
image
image

File: frontend/src/components/EnvironmentConfiguration.jsx [hunk 1]
```diff
@@ -756,11 +849,80 @@ export default function EnvironmentConfiguration (env) {
...
+                                        {label:'Latitude 3', key:'latitude3', type:'number'},
+                                        {label:'Longitude 3', key:'longitude3', type:'number'},
+                                        {label:'Latitude 4', key:'latitude4', type:'number'},
+                                        {label:'Longitude 4', key:'longitude4', type:'number'},
+                                    ].map((field, i) => (
+                                        <Grid item xs={6} key={i}>
+                                            <StyledInputLabel id={field.key}>{field.label}</StyledInputLabel>
+                                            <TextField
+                                                sx={{
+                                                    backgroundColor:'#71665E',
```
Comment (Anjali0407-git) [inline frontend/src/components/EnvironmentConfiguration.jsx:784]: @Hoshi-Okami The values I entered into the text box are not appearing. The text box remains empty.

File: frontend/src/components/EnvironmentConfiguration.jsx [hunk 2]
```diff
@@ -485,6 +506,78 @@ export default function EnvironmentConfiguration (env) {
...
+                Name:"SADE " + (sadeCount+1)
+            })
+        }
+
+        const popSade = () =>{
+            sadeArray.pop()
+        }
+        
+        const handleIncrement = () => {
+            setSadeCount(sadeCount +1)
```
Comment (Anjali0407-git) [inline frontend/src/components/EnvironmentConfiguration.jsx]: @Hoshi-Okami It seems increment button has some issue. When I clicked on increment  button once, multiple Sade zones added. You can look at [this demo](https://ooo.mmhmm.app/z_YCg9Rb1FL3ck8RkibbQp)
Make sure to pull latest code from frontend-updates for future work, drone screen has been updated in that.

**Review by Hoshi-Okami — COMMENTED** [review]
Fixed increment button and text box issues.

**Comment by Anjali0407-git** [comment]
@Hoshi-Okami The changes look good, and I am merging this PR with one follow-up task:

The list of Sade zones needs to be added to `mainJson `under the `Environment `property. For now, it's fine to store the list of Sade zones locally.
@ankyAgrawal I will assign this follow-up task to Valerie once @LaxminarayanaV7416's refactoring work is merged into frontend-updates. He has developed a great approach using class models, which makes our code easier to understand.

**Pull request merged**